spec/menu_spec.rb

```diff
@@ -0,0 +1,33 @@
+require 'menu'
+
+describe Menu do
+  let(:dish1) { double :dish1, name: "Dish 1", price: 5 }
+  let(:dish2) { double :dish2, name: "Dish 2", price: 10 }
+
+  it { is_expected.to respond_to(:menu_list) }
+  
+  it 'stores dishes to the menu' do
+    subject.add_dish(dish1)
+
+    expect(subject.menu_list).to include(dish1)
```

Comment:
I couldn't find a method or variable with the name menu_list so not sure what it's testing. Maybe the name of the method was changed?